atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -297,14 +297,14 @@ def _render_mail(self, rebuild, success, auto_canceled, manual_canceled):
         return (subject_template % formatting_dict, body_template % formatting_dict, log_files)
 
     def _send_mail(self, receivers_list, subject, body, log_files=None):
+        """Actually sends the mail with `subject` and `body` and optional log_file attachments
+        to all members of `receivers_list`."""
```

Comment:
This could be rephrased since it will be the docstring for the method

Reply:
I think it is a pretty decent docstring already, how would you like it rephrased? Maybe drop the "Actually"?

Reply:
> I think it is a pretty decent docstring already, how would you like it rephrased? Maybe drop the "Actually"?

Something in those lines (the 'actually' seems to be added there to refer to what was following next). Just a nitpick though.

Reply:
Ok, changed it